streaming/java/streaming-runtime/src/test/java/org/ray/streaming/runtime/resourcemanager/ResourceManagerTest.java

```diff
@@ -0,0 +1,103 @@
+package org.ray.streaming.runtime.resourcemanager;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.aeonbits.owner.util.Collections;
+import org.ray.api.Ray;
+import org.ray.streaming.jobgraph.JobGraph;
+import org.ray.streaming.runtime.BaseUnitTest;
+import org.ray.streaming.runtime.TestHelper;
+import org.ray.streaming.runtime.config.StreamingConfig;
+import org.ray.streaming.runtime.config.global.CommonConfig;
+import org.ray.streaming.runtime.config.master.ResourceConfig;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionGraph;
+import org.ray.streaming.runtime.core.master.resourcemanager.ResourceManager;
+import org.ray.streaming.runtime.core.master.resourcemanager.ResourceManagerImpl;
+import org.ray.streaming.runtime.core.master.scheduler.strategy.SlotAssignStrategy;
+import org.ray.streaming.runtime.core.master.scheduler.strategy.impl.PipelineFirstStrategy;
+import org.ray.streaming.runtime.core.resource.Container;
+import org.ray.streaming.runtime.core.resource.ContainerID;
+import org.ray.streaming.runtime.core.resource.Slot;
+import org.ray.streaming.runtime.graph.ExecutionGraphTest;
+import org.ray.streaming.runtime.master.JobRuntimeContext;
+import org.ray.streaming.runtime.master.graphmanager.GraphManager;
+import org.ray.streaming.runtime.master.graphmanager.GraphManagerImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ResourceManagerTest extends BaseUnitTest {
+
+  private static final Logger LOG = LoggerFactory.getLogger(ResourceManagerTest.class);
+
+  @org.testng.annotations.BeforeClass
+  public void setUp() {
+    LOG.warn("Do set up");
```

Comment:
Should this be an `info`?

Reply:
Also, should log more meaningful information, e.g., `"Setting up ResourceManagerTest."`

Reply:
This log has been deleted. Use ```BaseUnitTest``` to print the log.